atd-cr3-api/server.py

```diff
@@ -428,10 +428,11 @@ def user_delete_user(id):
 @APP.route("/events/", methods=["PUT", "POST"])
 def associate_location():
     # Require matching token
-    incoming_token = request.headers.get("HASURA_EVENT_API")
+    incoming_token = request.headers.get("HASURA_TRIGGER_API_KEY")
     incoming_event_name = request.headers.get("HASURA_EVENT_NAME", "")
+    environment_for_queue_name = API_ENVIRONMENT.lower()
     hashed_events_api = hashlib.md5()
-    hashed_events_api.update(str(HASURA_EVENT_API).encode("utf-8"))
+    hashed_events_api.update(str(HASURA_TRIGGER_API_KEY).encode("utf-8"))
     hashed_incoming_token = hashlib.md5()
     hashed_incoming_token.update(str(incoming_token).encode("utf-8"))
```

Comment:
Don't want to scope creep, but curious if anyone knows why we would md5 both values here to compare them? The value is a string. Could be overlooking something. 🤷

Reply:
this is a TIL for me—it appears to be an attempt to mitigate [short-circuiting](https://michaelhowardsecure.blog/2019/02/22/the-dangers-of-string-comparing-passwords/)

Reply:
that sounds right to me, @johnclary - a TIL for me as well. 🙏